sonaranalyzer-dotnet/its/expected/Nancy/Nancy-{34576216-0DCA-4B0F-A0DC-9075E75A676F}-S927.json

```diff
@@ -0,0 +1,69 @@
+{
+"issues":  [
+{
+"id":  "S927",
+"message":  "Rename parameter 'pipelineItem' to 'syncDelegate'.",
```

Comment:
While we're at the rule - perhaps tweak the message to give some context:
"Rename parameter '{0}' to '{1}' to match the inhertited parameter name."
(or something similar)
WDYT?